New version: PDAL_jll v2.8.2+0

jlbuild opened the pull request:
Autogenerated JLL package registration

* Registering JLL package PDAL_jll.jl
* Repository: https://github.com/JuliaBinaryWrappers/PDAL_jll.jl
* Version: v2.8.2+0
* Commit: b9db4529ff51ca90fade786d2ebf482827506160
* Revision on Yggdrasil: https://github.com/JuliaPackaging/Yggdrasil/commit/4e9b2e4dcd271fd57992fd5c9b694bc002cc0979
* Created by: @eschnett

giordano commented:
Loading error is real

[noblock]

eschnett commented:
I think a `LibCURL_jll` dependency is missing (is not declared). Could this be the reason?

[noblock]

giordano commented:
If the library requires libcurl directly then it should require the corresponding libcurl directly. Otherwise not. I don't know what's the situation with this package.

[noblock]

jlbuild committed:
New version: PDAL_jll v2.8.2+0

UUID: a8197b14-d70b-5660-b10f-8b1ebb62825c
Repo: https://github.com/JuliaBinaryWrappers/PDAL_jll.jl.git
Tree: d3358180822a2736f5562ede62b8178e1536e029

Registrator tree SHA: 191228b6dd8b9d0e2965ae3e705fe54c51dcfee8
